### PR TITLE
chore: Migrate EventServiceNoCompound to TS

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: src/script/event/EventServiceNoCompound.ts [hunk 1]
```diff
@@ -76,24 +87,24 @@ export class EventServiceNoCompound extends EventService {
...
-        const timestamp = new Date(record.time).getTime();
+        const timestamp = new Date(record.time);
         return (
           record.conversation === conversationId &&
           fromCompareFunction(fromDate, timestamp) &&
           toCompareFunction(toDate, timestamp)
         );
       })
-      .sort((a, b) => a.conversation - b.conversation)
+      .sort()
```
Comment (AndyLnd) [inline src/script/event/EventServiceNoCompound.ts]: As we are comparing strings here (conversationId) a comparator function is not necessary, as this is the default method for `sort()`

**Review by lipis — APPROVED** [review]

File: src/script/event/EventServiceNoCompound.ts (hunk 1 above)
Comment (ffflorian) [reply src/script/event/EventServiceNoCompound.ts]: But the `record` is of type `DBEvent` and not of type `string`?

**Review by ffflorian — CHANGES_REQUESTED** [review]

File: src/script/event/EventServiceNoCompound.ts (hunk 1 above)
Comment (AndyLnd) [reply src/script/event/EventServiceNoCompound.ts]: You are absolutely right. 🤦🏻‍♂️

**Review by ffflorian — APPROVED** [review]
Function return values would be nice, but it's okay without it for now.

**Pull request merged**